test/interpreter_functional/test_suites/run_pipeline/helpers.ts

```diff
@@ -110,13 +110,13 @@ export function expectExpressionProvider({
         stepContext: ExpressionValue = context
       ): Promise<ExpressionResult> => {
         log.debug(`running expression ${step || expression}`);
-        return browser.executeAsync<ExpressionResult>(
-          (
-            _expression: string,
-            _currentContext: ExpressionValue & { type: string },
-            _initialContext: ExpressionValue,
-            done: (expressionResult: ExpressionResult) => void
-          ) => {
+        return browser.executeAsync<
+          ExpressionResult,
```

Comment:
The problem with the current implementation that you cannot specify the first argument only, otherwise other arguments fall back to the default `unknown` value.

Reply:
Probably alright, even if it would be nice if TS was allowing that kind of 'partially defined, partially inferred' declaration.